### ISSUE TITLE
Cannot use super in loop - produces "Cannot read property 'body' of undefined"

### ISSUE THREAD

**Issue by dave-elkan** [6to5/6to5#530]
Imagine an implementation of EventEmitter which accepts an array of events for a single callback:

```javascript
var EventEmitter = require("events").EventEmitter;

class ExtendedEmitter extends EventEmitter {
  on(events, callback) {
    events.forEach(e => super.on.call(this, e, callback));
  }
}
```
This unfortunately produces this error:
```
TypeError: ExendedEmitter.js: Cannot read property 'body' of undefined
    at ReplaceSupers.getThisReference (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/transformation/helpers/replace-supers.js:155:25)
    at getThisReference (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/transformation/helpers/replace-supers.js:135:23)
    at ReplaceSupers.specHandle (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/transformation/helpers/replace-supers.js:228:23)
    at traverse.enter (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/transformation/helpers/replace-supers.js:141:23)
    at TraversalContext.enterNode (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/traverse/index.js:67:16)
    at TraversalContext.visitNode (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/traverse/index.js:116:15)
    at TraversalContext.visit (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/traverse/index.js:131:17)
    at traverseNode (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/traverse/index.js:159:17)
    at TraversalContext.visitNode (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/traverse/index.js:120:3)
    at TraversalContext.visit (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/traverse/index.js:139:26)
    at traverseNode (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/traverse/index.js:159:17)
    at TraversalContext.visitNode (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/traverse/index.js:120:3)
    at TraversalContext.visit (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/traverse/index.js:131:17)
    at traverseNode (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/traverse/index.js:159:17)
    at traverse (/Users/dave/.nvm/v0.10.30/lib/node_modules/6to5/lib/6to5/traverse/index.js:179:5)
```
This format works around the issue
```javascript
var EventEmitter = require("events").EventEmitter;

class ExtendedEmitter extends EventEmitter {
  on(events, callback) {
    var superOn = super.on;
    events.forEach(e => superOn.call(this, e, callback));
  }
}
```

**Comment by sebmck** [comment]
Fixed as of 2.13.2, thanks!

**Comment by dave-elkan** [comment]
Thanks @sebmck!
